### ISSUE TITLE
MM extension shows wrong number of tokens after token re-denomination. How to fix?

### ISSUE THREAD

**Issue by wzup** [MetaMask/metamask-extension#10669]
**Describe the bug**
There was a [1:10 token re-denomination](https://btcst.medium.com/btcst-redenomination-guide-for-exchange-and-wallet-partners-ad00eade8cec). But token balance in MM hasn't updated. Amount of tokens still remains the same as before the re-domination.

**Steps to reproduce (REQUIRED)**

1. Read about [token re-denomination](https://btcst.medium.com/btcst-redenomination-guide-for-exchange-and-wallet-partners-ad00eade8cec).
2. You should have BTCST token in your Metamask wallet before 17 March

**Expected behavior**
After 18 March the amount of tokens had to be updated 1:10 ratio. For every single token you have to have 10 tokens. The decimal increased from 16 to 17.

**Screenshots**
Decimals changed [from 16 to 17 in token contract](https://bscscan.com/address/0x78650b139471520656b9e7aa7a5e9276814a38e9#readProxyContract). But Metamask extension hasn't updated it.
![image](https://user-images.githubusercontent.com/3880497/111618855-e05a5900-880a-11eb-9f82-775fc135f469.png)

**Browser details (please complete the following information):**
 - OS: OS X
 - Hardware Wallet: --
 - Browser: Version 89.0.4389.90 (Official Build) (x86_64)
 - MetaMask Version: 9.2.0

**Additional context (Error Messages, etc.)**
1. Steps in this article doesn't fix the issue 
https://metamask.zendesk.com/hc/en-us/articles/360028059272-What-to-do-when-your-balance-of-ETH-and-or-ERC20-tokens-is-incorrect-inaccurate
2. There is NO option to _re-add_ or change token settings in Metamask.

![image](https://user-images.githubusercontent.com/3880497/111618454-66c26b00-880a-11eb-9ba2-3d6e77c260b1.png)

**Comment by tmashuang** [comment]
What if you hide the token first then re-add it?

**Comment by wzup** [comment]
> What if you hide the token first then re-add it?

Thanks. It worked for me.

However. I had to withdraw all my tokens before this steps. I had to stake my token with their app because their app displayed correct balance.
So I had zero balance when I was doing steps below. IDK if it is ok with full balance.

1. Select token and click token settings. Then "Hide token"
![image](https://user-images.githubusercontent.com/3880497/111646185-713f2d80-8827-11eb-9ef6-33fa4a57c460.png)

2. Then add the token as a custom token back again. Make sure that the "Decimals of Precision" is correct.
![image](https://user-images.githubusercontent.com/3880497/111647179-46090e00-8828-11eb-869e-38d55c7c94df.png)
